pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -23,6 +23,24 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// DefaultPreemptionArgs holds arguments used to configure the
+// DefaultPreemption plugin.
+type DefaultPreemptionArgs struct {
+	metav1.TypeMeta
+
+	// MaxCandidateNodesPercentage is the maximum number of candidates to
+	// shortlist when dry running preemption as a percentage of number of nodes.
+	MaxCandidateNodesPercentage int32
+	// MinCandidates is the minimum number of candidates to shortlist. The
+	// likely number of candidates enumerated for dry running preemption is given
+	// by the formula max(NumNodes * MaxCandidateNodesPercentage, MinCandidates).
```

Comment:
Since the goal is to reduce the number of candidates, why not use the formula `min(NumNodes * MaxCandidateNodesPercentage, MinCandidates)`?

If we use `max(...)`,  will `MinCandidateNodesPercentage` be a better name?

Reply:
The goal is to reduce the number of candidates in a controlled way. `max` seems safer.

`MinCandidateNodesPercentage` is actually more accurate, given that we won't stop until we find at least one Node where PDB is not violated.

Reply:
Good catch, changed the two arg names to `minCandidateNodesPercentage` and `minCandidateNodesAbsolute`. Should be much more self-explanatory now.